Makefile

```diff
@@ -79,10 +83,16 @@ ifndef CUQUANTUM_SDK
 	@test $(CUQUANTUM_SDK)
 endif
 	rm -rf ./BuildTests
-	cmake . -BBuildTests -DBUILD_TESTS=1 -DPLLGPU_BUILD_TESTS=1 -DCUQUANTUM_SDK=$(CUQUANTUM_SDK)
+	cmake . -BBuildTests -DBUILD_TESTS=1 -DPLLGPU_BUILD_TESTS=1 -DCUQUANTUM_SDK=$(CUQUANTUM_SDK) -DPLLGPU_ENABLE_MPI=OFF
```

Comment:
I think we should avoid setting things to their default value. This makes the code more verbose.

Reply:
Thanks! Removed.